docs/merkletree/treetex/README.md

```diff
@@ -0,0 +1,79 @@
+Treetex
+=======
+
+`treetex` is a tool for drawing/visualising Merkle trees.
+
+Ensure you have latex installed on your system if you want to this; the tool
```

Comment:
```suggestion
Ensure you have `latex` command installed on your system if you want to use this; the tool
```
Also, rewrap after the modification.

Reply:
You need more than just the command installed, there's a lot of support files etc. too.

Reply:
SGTM.